docs/knowledgebase/advanced/try-runtime.md

```diff
@@ -0,0 +1,184 @@
+---
+title: try-runtime
+---
+
+The `try-runtime` tool is built to query a snapshot of runtime storage, using an in-memory-externalities to store state. In this way, 
+it enables runtime engineers to write tests for a specified runtime state.
+Being part of [`sc-cli`][sc-cli-rustdocs], it is designed to be used as a command line interface to specify 
+what block to capture and which pallets to run tests on.
```

Comment:
```suggestion
Being part of [`sc-cli`][sc-cli-rustdocs], it is designed to be used as a command line interface to specify at which block to query state.
```
Perhaps beyond scope, but could also be worth adding that when we query at a bock height we get the state after the block is applied against it with the given runtime.